SCRAM/GMake/Makefile.rules

```diff
@@ -1136,14 +1136,15 @@ $(eval $(call LogFile,$(1),$(2)))
 .PHONY: $(1) all_$(1)
 $(1)_objdir := $(WORKINGDIR)/$(2)/$(1)
 $(1)_srcdir       := $(LOCALTOP)/$(2)
-ifneq ($(strip $(CUDASRC_FILES_SUFFIXES)),)
 $(1)_cudafiles    := $$(filter $(foreach e,$(CUDASRC_FILES_SUFFIXES),%.$e),$($(1)_files))
+ifneq ($(strip $$($(1)_cudafiles)),)
```

Comment:
`CUDASRC_FILES_SUFFIXES` is always defined, this way `$(1)_cudaobj` is defined only if there is any .cu files

Reply:
you are right, CUDASRC_FILES_SUFFIXES is always set